### DISCUSSION TITLE
how to draw a block matrix like this

### DISCUSSION THREAD

**Discussion by kzqj** [typst/typst#5207] [Q&A] [answered]
how to draw a block matrix like this, ignore the blue line
![QQ20241015-111440](https://github.com/user-attachments/assets/57824a44-0cc7-47a4-a360-1e27fb57b3cf)
![QQ20241015-110139](https://github.com/user-attachments/assets/677ac464-9f2f-4d33-8552-50d00731623e)

**Comment by ruben-hude** [comment, marked as answer]
You can do something like that:
```typst
$ mat(
    -1, display(mat(delim: #none, -1, 3, -1));
    display(mat(delim: #none, 0;0;0)), A_3) $
```
